public/app/features/admin/UserProfile.tsx

```diff
@@ -265,13 +265,13 @@ export class UserProfileRow extends PureComponent<UserProfileRowProps, UserProfi
         <td className={labelClass}>{label}</td>
         <td className="width-25" colSpan={2}>
           {this.state.editing ? (
-            <Input
-              className="width-20"
+            <Forms.Input
+              size="md"
               type={inputType}
               defaultValue={value}
               onBlur={this.onInputBlur}
               onChange={this.onInputChange}
-              inputRef={this.setInputElem}
+              ref={this.setInputElem}
```

Comment:
So, this should be not necessary for focusing the input, as the input is rendered when the state is `editing` and you can use autofocus property